.travis.yml

```diff
@@ -37,7 +37,7 @@ before_install:
   - composer self-update
   - composer require symfony/symfony:${SYMFONY_VERSION} --no-update
   
-install: composer update --prefer-source $COMPOSER_FLAGS
+install: composer update -vvv --prefer-source $COMPOSER_FLAGS
```

Comment:
no luck. can we do something to reduce the options space? like not allow symfony > 2.3 in the lowest build?